### ISSUE TITLE
Blocked on "Open source project"

### ISSUE THREAD

**Issue by gautier0713** [mtaheij/PoGo-UWP#66]
I have a problem, my Lumia 930 is on Windows 10, in developer mode, I can install PoGo (latest version), I can download the app and all but, when I try to run it, it stays like 20 seconds on the black screen with "Open source project" and the symbol written and then switch off and go back to my apps page... Someone knows something ?

**Comment by DarkLegend481** [comment]
Are you using which version of Windows 10? If 10586, you need a update to 14393, at least, to play PoGo

**Comment by gautier0713** [comment]
Upgrade done ! Thank you very much for your help ! But now is another issue: I cannot connect to the game even if I know I am entering the right username and password of my PTC account ! Do you know why ? I'm stuck on the connection page and it does nothing when I click the button to connect...

**Comment by DarkLegend481** [comment]
Did you tried uninstall PoGo and install it again?
